src/pages/Home/index.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import IntroSection from './IntroSection';
 import GoalsSection from './GoalsSection';
+import Header from '../../components/Header';
 
 function Home() {
   return (
     <>
+      <Header />
       <IntroSection />
       <GoalsSection />
```

Comment:
Thank you, @danielufeli, for implementing the home page.  I think everything is good at this moment. You can merge it.